integration_test.go

```diff
@@ -39,70 +39,104 @@ func TestIntegrationCartographer(t *testing.T) {
-			online:      false,
-			imuEnabled:  false,
-			mode:        cartofacade.MappingMode,
-			subAlgo:     viamcartographer.Dim2d,
+			description:     "offline mapping mode 2D with lidar only",
+			online:          false,
+			imuEnabled:      false,
+			odometerEnabled: false,
+			mode:            cartofacade.MappingMode,
+			subAlgo:         viamcartographer.Dim2d,
 		},
 		{
-			description: "offline sensor updating mode 2D",
-			online:      false,
-			imuEnabled:  false,
-			mode:        cartofacade.UpdatingMode,
-			subAlgo:     viamcartographer.Dim2d,
+			description:     "offline updating mode 2D with lidar only",
+			online:          false,
+			imuEnabled:      false,
+			odometerEnabled: false,
+			mode:            cartofacade.UpdatingMode,
+			subAlgo:         viamcartographer.Dim2d,
 		},
 		// Online run with lidar + imu
 		{
-			description: "online with imu sensor mapping mode 2D",
-			online:      true,
-			imuEnabled:  true,
-			mode:        cartofacade.MappingMode,
-			subAlgo:     viamcartographer.Dim2d,
+			description:     "online mapping mode 2D with lidar + imu",
+			online:          true,
+			imuEnabled:      true,
+			odometerEnabled: false,
+			mode:            cartofacade.MappingMode,
+			subAlgo:         viamcartographer.Dim2d,
 		},
 		{
-			description: "online with imu sensor localizing mode 2D",
-			online:      true,
-			imuEnabled:  true,
-			mode:        cartofacade.LocalizingMode,
-			subAlgo:     viamcartographer.Dim2d,
+			description:     "online localizing mode 2D with lidar + imu",
+			online:          true,
+			imuEnabled:      true,
+			odometerEnabled: false,
+			mode:            cartofacade.LocalizingMode,
+			subAlgo:         viamcartographer.Dim2d,
 		},
 		{
-			description: "online with imu sensor updating mode 2D",
-			online:      true,
-			imuEnabled:  true,
-			mode:        cartofacade.UpdatingMode,
-			subAlgo:     viamcartographer.Dim2d,
+			description:     "online updating mode 2D with lidar + imu",
+			online:          true,
+			imuEnabled:      true,
+			odometerEnabled: false,
+			mode:            cartofacade.UpdatingMode,
+			subAlgo:         viamcartographer.Dim2d,
+		},
+		// Online run with lidar + odometer
+		{
+			description:     "online mapping mode 2D with lidar + odometer",
+			online:          true,
+			imuEnabled:      false,
+			odometerEnabled: true,
+			mode:            cartofacade.MappingMode,
+			subAlgo:         viamcartographer.Dim2d,
+		},
+		{
+			description:     "online localizing mode 2D with lidar + odometer",
+			online:          true,
+			imuEnabled:      false,
+			odometerEnabled: true,
+			mode:            cartofacade.LocalizingMode,
+			subAlgo:         viamcartographer.Dim2d,
+		},
+		{
+			description:     "online updating mode 2D with lidar + odometer",
+			online:          true,
+			imuEnabled:      false,
+			odometerEnabled: true,
+			mode:            cartofacade.UpdatingMode,
+			subAlgo:         viamcartographer.Dim2d,
```

Comment:
I think it would be beneficial to have a case where both odom and imu data are being used

Reply:
It was decided that it would be out-of-scope for the ticket.